package.json

```diff
@@ -29,13 +29,13 @@
   },
   "author": "Felix Becker <felix@sourcegraph.com>",
   "dependencies": {
-    "@sourcegraph/event-positions": "0.0.1",
+    "@sourcegraph/event-positions": "^1.0.0",
     "@sourcegraph/react-loading-spinner": "0.0.6",
     "highlight.js": "^9.12.0",
     "lodash": "^4.17.10",
     "marked": "^0.4.0",
     "mdi-react": "^4.0.0",
-    "rxjs": "^6.2.1",
+    "rxjs": "^6.3.2",
```

Comment:
This release was already included in the previous range, was 6.2.1 actually not compatible with codeintellify anymore? In general it's good to have constraints be wide so consumers don't duplicate packages if not needed, i.e. only update package-lock.json in this repo as a chore (Renovate will do that) unless of course we depend on a new feature from a new version.

Reply:
I was seeing the “source” rxjs type error where one of the types had codeintellify in the path. I think it may have been unnecessary, but upgrading this and some other packages’ rxjs ended up fixing it.